deployment_jobs/Create_Model_and_Deployment_Job.py

```diff
@@ -4,6 +4,7 @@
 
 # COMMAND ----------
 
+# MAGIC %pip uninstall -y mlflow-skinny
 # MAGIC %pip install git+https://github.com/mlflow/mlflow.git@mlflow-3
```

Comment:
can we put these in the same line? i think that is preferred so pip can resolve it all at once

Reply:
Can you install and uninstall in the same command?